components/has/.tekton/trigger-template.yaml

```diff
@@ -31,12 +31,6 @@ spec:
           name: appstudio-dockerfile-builder
         workspaces:
           - name: workspace
-            volumeClaimTemplate:
-              spec:
-                accessModes:
-                  - ReadWriteOnce
-                resources:
-                  requests:
-                    storage: 10Mi
-                #storageClassName: gp2
-                volumeMode: Filesystem
\ No newline at end of file
+            persistentVolumeClaim:
+              claimName: app-studio-default-workspace
+            subPath: application-service-$(tt.params.git-revision)
```

Comment:
Can we do `application-service/$(tt.params.git-revision)`

Reply:
though we can leave it as is since this workspace only has a single component anyway.